singularity-eos/eos/eos_base.hpp

```diff
@@ -68,6 +68,7 @@ char *StrCat(char *destination, const char *source) {
   using EosBase<EOSDERIVED>::InternalEnergyFromDensityTemperature;                       \
   using EosBase<EOSDERIVED>::PressureFromDensityTemperature;                             \
   using EosBase<EOSDERIVED>::PressureFromDensityInternalEnergy;                          \
+  using EosBase<EOSDERIVED>::MinInternalEnergyFromDensity;                               \
```

Comment:
Still needs to be added to the Python interface, but that can be its own PR.

Reply:
Issue here: https://github.com/lanl/singularity-eos/issues/321